core/arch/arm/kernel/secure_partition.c

```diff
@@ -352,6 +380,22 @@ static TEE_Result sp_dt_get_u32(const void *fdt, int node, const char *property,
 	return TEE_SUCCESS;
 }
 
+static TEE_Result sp_dt_get_u16(const void *fdt, int node, const char *property,
+				uint16_t *value)
+{
+	uint32_t dt_value;
+	TEE_Result res = TEE_SUCCESS;
+
+	res = sp_dt_get_u32(fdt, node, property, &dt_value);
```

Comment:
This is not going to work since it's supposed to be a 16-bit value and `sp_dt_get_u32()` errors out if it's not a 32-bit value.

Reply:
Corrected